src/TFTManager.cpp

```
@@ -772,21 +808,21 @@ void TFTManager::renderTextLandscape(const char* l1, const char* l2, uint32_t l1
             full.deleteSprite();
-            return;
+            return true;
         }
         Serial.println("TFT: PSRAM text sprite failed, using strips");
     }
     const int STRIP_H = 32;
-    LGFX_Sprite strip(&_tft);
-    strip.setColorDepth(16);
-    if (!strip.createSprite(W, STRIP_H)) return;
+    _strip.setColorDepth(16);
+    if (!_strip.createSprite(W, STRIP_H)) return false;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)